apps/shell/tash_command.c

```diff
@@ -269,6 +269,50 @@ int tash_execute_cmd(char **args, int argc)
 	return 0;					/* Need to pass the appropriate error later */
 }
 
+bool tash_tab_cmd(char *cmd, int *pos, bool pressed_tab)
+{
+	int cmd_idx;
+	int found_cnt = 0;
+	int idx = -1;
+	/* lock mutex */
+	pthread_mutex_lock(&tash_cmds_info.tmutex);
```

Comment:
How about moving this below the declaration of search array?

Reply:
done.